src/test/java/core/basesyntax/service/impl/FileReadImplTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileRead;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FileReadImplTest {
+    private FileRead fileRead;
+    private File tempFile;
+
+    @BeforeEach
+    public void setUp() throws IOException {
+        fileRead = new FileReadImpl();
+        tempFile = File.createTempFile("testData", ".txt");
+        tempFile.deleteOnExit();
```

Comment:
The call to 'deleteOnExit()' is not needed as you are manually deleting the file in the tearDown method.